src/Sulu/Bundle/WebsiteBundle/Resources/config/services.xml

```diff
@@ -177,11 +177,20 @@
         </service>
 
         <!-- exception controller -->
-        <service id="sulu_website.exception.controller" class="%sulu_website.exception.controller.class%">
+        <service decorates="twig.controller.exception" class="%sulu_website.exception.controller.class%" public="false">
```

Comment:
this change will enhance to remove following lines:

* https://github.com/sulu/sulu-minimal/blob/master/app/config/website/config.yml#L7
* https://github.com/sulu/sulu-standard/blob/develop/app/config/website/sulu.yml#L9